lib/normalize_title_filter.js

```diff
@@ -94,7 +66,8 @@ module.exports = function(hyper, req, next, options, specInfo) {
                 // until https://phabricator.wikimedia.org/T130757 (VE
                 // redirect support) is resolved.
                 // TODO: Remove.
-                && /^WikipediaApp\//.test(req.headers['user-agent'])) {
+                && /^WikipediaApp\//.test(req.headers['user-agent'])
+                && req.query.redirect !== 'no') {
```

Comment:
Lets declare this as an official parameter, and rely on the coercion code in hyperswitch to normalize this (see https://github.com/wikimedia/hyperswitch/pull/33).